v2/src/modules/address/ModuleAllEthTransfers.vue

```diff
@@ -1,12 +1,12 @@
 <template>
     <template v-if="!xs">
         <v-row align="center" justify="start" class="text-info mt-2 mt-sm-3">
-            <v-col sm="4" md="3" lg="2" order="1"> Status </v-col>
-            <v-col sm="5" md="3" lg="2" order="2"> Address </v-col>
-            <v-col sm="3" lg="2" order-sm="last" order-lg="3"> Value </v-col>
-            <v-col lg="2" class="d-none d-lg-flex" order="4"> Tx Fee Paid </v-col>
-            <v-col sm="3" class="d-none d-lg-flex" lg="2" order="5"> Type </v-col>
-            <v-col md="3" lg="2" class="d-none d-md-flex" order-md="3" order-lg="6"> Hash/Block </v-col>
+            <v-col sm="4" md="3" lg="2" order="1"> {{ $t('common.status') }} </v-col>
+            <v-col sm="5" md="3" lg="2" order="2"> {{ $t('common.address') }}s </v-col>
+            <v-col sm="3" lg="2" order-sm="last" order-lg="3"> {{ $t('common.value') }} </v-col>
+            <v-col lg="2" class="d-none d-lg-flex" order="4"> {{ $t('common.txFee') }} {{ $t('common.paid') }} </v-col>
```

Comment:
please create a new whole string 'Tx Fee Paid' the string will break when translated to a different language